GOVERNANCE.md

```diff
@@ -4,51 +4,170 @@ This document defines governance policies for the Kiali project.
+
+### Becoming a Leader
+
+To become a Leader, you must first be a Maintainer. Then, a new Leader must be proposed
+by an existing Maintainer by opening a pull request modifying the leaders list to add the new
+proposed leader, or via a by opening a [GitHub discussion](https://github.com/kiali/kiali/discussions/new)
+under the Governance category. The following information must be provided:
+
+* nominee's GitHub username,
+* an explanation of why the nominee should be a leader.
+
+Leadership is granted if a simple majority vote among current maintainers succeeds.
+
+## Inactivity
+
+It is important for contributors to be and stay active to set an example and show commitment to the project. Inactivity is harmful to the project as it may lead to unexpected delays, contributor attrition, and a lost of trust in the project.
+
+* Inactivity is measured by:
+    * Periods of no contributions for longer than 4 months
+    * Periods of no communication for longer than 4 months
+* Consequences of being inactive include:
+    * Involuntary removal or demotion
+    * Being asked to move to Emeritus status
+
+## Involuntary Removal or Demotion
+
+Involuntary removal/demotion of a contributor happens when requirements for a certain role aren't being met. This may include repeated patterns of inactivity, extended period of inactivity, a period of failing to meet the requirements of your role, and/or a violation of the Code of Conduct. This process is important because it protects the community and its deliverables while also opens up opportunities for new contributors to step in.
+
+Involuntary removal or demotion is handled through a vote by simple majority of the current Maintainers.
+
+## Stepping Down/Emeritus Process
+
+If and when contributors commitment levels change, contributors can consider stepping down (moving down the contributor ladder) vs moving to emeritus status (completely stepping away from the project).
 
-Maintainers can be removed by a 2/3 majority vote by maintainers, or by resignation.
+Contact the Maintainers about changing to Emeritus status, or reducing your contributor level.
 
-## GitHub Project Administration
+## Voting
 
-Maintainers will be added to the GitHub @kiali/maintainers team, and made a GitHub maintainer of that team.
-They will be given write permission to the Kiali GitHub repositories https://github.com/kiali.
+While most business in Kiali project is conducted by "lazy consensus", periodically
+the Maintainers may need to vote on specific actions or changes.
+A vote can be taken on pull requests proposing project changes, or via a
+[a GitHub discussion](https://github.com/kiali/kiali/discussions/new).
+If there are security or conduct matters, voting can take place in private meetings.
+Any Maintainer or Tester may demand a vote be taken.
 
-## Changes in Governance
+Most votes require a simple majority of all Maintainers to succeed. Changes
+to the Governance require a 2/3 vote of all Maintainers.
```

Comment:
What if not all maintainers are available?  Can a vote be made with a quorum of maintainers, some % or minimum number of maintainers?  Can a maintainer officially abstain?  Should votes be time-boxed such that votes don't wait indefinitely for maintainers that maybe away or hard to reach?

Reply:
I remember this was mentioned in the discussion thread (#4199).

I thought a lot on this and I chose to left it unchanged (same policy as in the past). My current rationale is that if there is one or two maintainers unavailable, simple majority (or 2/3 vote) can still be achieved and that would mean that even if unavailable people would have voted against, that wouldn't changed the outcome.

Of course, there can be cases where several maintainers are unavailable, and that happened very recently. I had the idea that the team preference would be to wait until people become available before doing important decisions and, IMHO, this is a generous way to proceed.

About time-boxing voting, I think that would be a good idea. Perhaps limit time to 5 working days??? (which is the same limit as when proposing new maintainers).

About abstention, I would say it shouldn't be allowed. "Commitment to the project" is a pledge of a maintainer, and I think abstentions demonstrate the contrary. Anyway, perhaps can we use this PR to see how much engagement we can expect from people?? If we can't reach that 2/3 because of abstentions, we should really consider adding a paragraph about how to deal with abstentions. So far, I see some people reviewing and discussing, which is good. But I don't see any approval yet :-/

Reply:
Yes, it's fine, we don't need to "over-engineer" the voting unless we actually end up with a problem, and then we can make an amendment.  I guess it was the "2/3 vote of all Maintainers" that made me think that all maintainers had to vote before it was official.  What you are saying here is that the voting can stop as soon as approval or rejection is mathematically secured.

Time-boxing of 5 days sounds reasonable to me.

Reply:
I think it's necessary a minor comment about when an important voting is taking place.
Like that it will give time to maintainers to attend.
To avoid situations that there is a "voting" for "tomorrow" when some folks are in PTO or any other absence.

Reply:
I agree with @lucasponce comment. We have to avoid the fire-fighting (again) and do things in more cruise speed.

Reply:
@lucasponce @xeviknal IMHO, current wording gives this flexibility. There is no statement about how to start a vote. So, I think it's in our hands to be kind with the team and start when it's most appropriate for everybody.

However, as suggested by Jay, once the vote starts it should be time-boxed. I'm going to add this sentence in a moment.

Reply:
I would limit to more than 5 working days. Even for the reason that working days might be different from country to country (say Israel..). 7 might cover more cases and make this more inclusive.

Reply:
I am OK with the update to 5 days in the doc, although as @abonas states, 7 days may be more reasanable.  I'm resolving this conversation as I'll be fine either way.

Reply:
I modified to 7 days